src/components/staking/ModalWithdrawRewards.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <ActionModal
+    id="modal-withdraw-rewards"
+    ref="actionModal"
+    :submit-fn="submitForm"
+    :simulate-fn="simulateForm"
+    title="Withdraw"
+    class="modal-withdraw-rewards"
+    submission-error-prefix="Withdrawal failed"
+  >
+    <TmFormGroup
+      class="action-modal-form-group"
+      field-id="amount"
+      field-label="Amount"
+    >
+      <span class="input-suffix">{{ denom | viewDenom }}</span>
+      <TmField id="amount" :value="rewards | atoms | fullDecimals" readonly />
+    </TmFormGroup>
+    <span v-if="!validatorAddress" class="form-message">
```

Comment:
should the message show always or only when the user actually has more then 5 delegations?

Reply:
🤦‍♂ You're right!

Reply:
Actually, is it necessary? We could leave it there to inform the user regardless?

Reply:
I personally like to reduce noise that is not important to users. @jbibla preferences?

Reply:
* It would have a minor impact on the message wording. The current phrasing is generic.